enderio-base/src/main/java/com/enderio/base/data/recipe/standard/EIOItemTagsProvider.java

```diff
@@ -0,0 +1,26 @@
+package com.enderio.base.data.recipe.standard;
+
+import com.enderio.base.EnderIO;
+import com.enderio.base.common.tag.EIOTags;
+
+import net.minecraft.data.DataGenerator;
+import net.minecraft.data.tags.BlockTagsProvider;
+import net.minecraft.data.tags.ItemTagsProvider;
+import net.minecraftforge.common.Tags;
+import net.minecraftforge.common.data.ExistingFileHelper;
+
+public class EIOItemTagsProvider extends ItemTagsProvider{
+    
+    public EIOItemTagsProvider(DataGenerator pGenerator, BlockTagsProvider pBlockTagsProvider,
+            ExistingFileHelper existingFileHelper) {
+        super(pGenerator, pBlockTagsProvider, EnderIO.MODID, existingFileHelper);
+    }
+
+    @Override
+    protected void addTags() {
+        copy(EIOTags.Blocks.CLEAR_GLASS, EIOTags.Items.CLEAR_GLASS);
```

Comment:
Not personally a fan of these copies, can you add the tag in the item builder (anytime after item() but before build() instead?

Reply:
I did that originally, but this is how vanilla handles it. Blocks have BlockTags which are converted to ItemTags in this way. But I can Change it back to assign an Itemtag instead of copy.

Reply:
Think I'd personally prefer it to be with the blocks, but do as you please :)

Reply:
I think the usage of copy is better, because it's more in line with vanilla